ng-dev/auth/shared/node-handler.ts

```diff
@@ -0,0 +1,159 @@
+/*
+ * Copyright 2017 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the
+ * License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {EventEmitter} from 'events';
+import * as Http from 'http';
+import * as Url from 'url';
+
+import {AuthorizationRequest} from '@openid/appauth';
+import {AuthorizationRequestHandler, AuthorizationRequestResponse} from '@openid/appauth';
+import {AuthorizationError, AuthorizationResponse} from '@openid/appauth';
+import {AuthorizationServiceConfiguration} from '@openid/appauth';
+import {Crypto} from '@openid/appauth';
+import {log} from '@openid/appauth';
+import {BasicQueryStringUtils, QueryStringUtils} from '@openid/appauth';
+import {NodeCrypto} from '@openid/appauth/built/node_support/crypto_utils.js';
+
+// TypeScript typings for `opener` are not correct and do not export it as module
+import opener from 'opener';
```

Comment:
What's wrong with this import? it's just a function exported through the default export right?